kroxylicious-filters/kroxylicious-encryption/src/test/resources/serialization/parcel/header-serialization-disabled.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
I think we've mostly been favouring Yaml over json for test resources.  We should be consistent unless there's a more reason.